promgen/notification/slack.py

```diff
@@ -45,5 +53,11 @@ def _send(self, url, data):
         json = {
             'text': message,
         }
-
-        util.post(url, json=json).raise_for_status()
+         
+        if proxy is not None and proxy != "":
+            logger.info('Notifying Slack using proxy')
+            util.post(url, json=json, proxies=proxies).raise_for_status()
+        else:
+            logger.info('Notifying Slack without proxy')
+            util.post(url, json=json).raise_for_status()
+        
```

Comment:
```suggestion
         
        util.post(url, json=json, **kwargs).raise_for_status()
```

Reply:
Thanks for the advice, actually it looks way better using the kwargs, we will update the code, test it and update the pull request, thanks a lot.